packages/fluentui/react-bindings/src/hooks/useStyles.ts

```diff
@@ -9,12 +9,12 @@ import {
 import * as React from 'react';
 // @ts-ignore We have this export in package, but it is not present in typings
 import { ThemeContext } from 'react-fela';
```

Comment:
Oops, should we still have `ThemeContext` from `react-fela` here?

Reply:
Yep, as we are receiving values from there. This will be a next and the last step of decoupling as we are using this context in our components, too.